static/events/date_picker.js

```diff
@@ -0,0 +1,43 @@
+const dateField = document.getElementById('date-picker')
+dateField.addEventListener("click", openDatePicker())
+
+const clearAll = document.getElementById('clear-filters')
+clearAll.addEventListener("click",function() {clearFilters()})
+
+function openDatePicker(){
+    new Litepicker({
+    element: document.getElementById('date-picker'),
+    singleMode: false,
+    setup: (picker) => {
+
+      picker.on('selected', () => {
+        updateDateInForm()
+        
+      });
+    },
+})
+
+  }
+
+
+function updateDateInForm(){
+  const date_value = document.getElementById('date-picker').value
+  const start_date_value = date_value.slice(0,10)
+  const end_date_value = date_value.slice(13)
+  const start_time_field = document.getElementById('id_start_from_to_time_0')
+  const end_time_field = document.getElementById('id_start_from_to_time_1')
+  start_time_field.value = start_date_value
+  if(start_date_value != end_date_value){
+    end_time_field.value = end_date_value
+  }
+}
```

Comment:
Selected date should be passed to this method.
This method should only update date in the argument to element